AWS-CodePipeline/sam-monorepo-pipeline-template/questions.json

```diff
@@ -0,0 +1,214 @@
+{
+    "questions": [
+        {
+            "question": "What is subfolder path of your project from the root of the repository?",
```

Comment:
Without a default value, I think it is worth to add an example here. When I am asked this question, I need to decide if I need to use `/sub-folder`, `sub-folder` or `./sub-folder`.